### ISSUE TITLE
10.0.4 Storage Dropbox is duplicate present

### ISSUE THREAD

**Issue by mmattel** [owncloud/core#29808]
### Steps to reproduce
1. run sudo -uwww-data ./occ files_external:backends
2. same in the browser under settings/storage

### Expected behaviour
Dropbox should only be visible once (oauth1 is no longer supported by Dropbox)

### Actual behaviour
```
- dropbox:
      - name: Dropbox
      - identifier: dropbox
      - configuration:
      - storage_class: \OCA\Files_External\Lib\Storage\Dropbox
      - supported_authentication_backends:
        - oauth1::oauth1
      - authentication_configuration:
        - oauth1::oauth1:
          - app_key: text
          - app_secret: password

- files_external_dropbox:
    - name: Dropbox
    - identifier: files_external_dropbox
    - configuration:
    - storage_class: \OCA\Files_external_dropbox\Storage\Dropbox
    - supported_authentication_backends:
     - oauth2::oauth2
    - authentication_configuration:
      - oauth2::oauth2:
        - client_id: text
        - client_secret: password
```

![pastedgraphic-1](https://user-images.githubusercontent.com/3321281/33809544-e34ec322-ddf8-11e7-85d5-d86f6460e227.png)

### Server configuration
**Operating system**: Ubuntu 16.04

**Web server:** nginx

**Database:** mysql

**PHP version:** 7

**ownCloud version:** (see ownCloud admin page) 10.0.4

**Updated from an older ownCloud or fresh install:** updated

**Where did you install ownCloud from:** tar

**Signing status (ownCloud 9.0 and above):**

```
No issues
```

(I removed all the rest as it delivers no additional value)

**Comment by ownclouders** [comment]
[GitMate.io](https://gitmate.io) thinks the contributor most likely able to help you is @PVince81.

**Comment by kdslkdsaldsal** [comment]
https://github.com/owncloud/core/pull/29135 was merged after the 10.0.4 release so the "old" Dropbox entry will be gone in 10.0.5

**Comment by mmattel** [comment]
Thx, bad - thought it made it in... 
(It looks really sh...y)
